tests/Collection/CrudSpecFunctionalTest.php

```diff
@@ -19,9 +20,11 @@
  */
 class CrudSpecFunctionalTest extends FunctionalTestCase
 {
+    use SetUpTearDownTrait;
+
     private $expectedCollection;
 
-    public function setUp()
+    private function doSetUp()
```

Comment:
Just to confirm, can we still chain to `parent::setUp()` below despite this method being renamed to `doSetUp()`?

Reply:
Yes, that is still possible. The `parent::setUp()` call in `doSetUp` will always go to the parent class. In this case, the parent (`FunctionalTestCase`) also includes the trait, so the call will again land in the shim and call the `doSetUp` method that is written in `FunctionalTestCase`. Chaining the setup/teardown methods is still possible as before, we just have to remember to use the polyfill trait and write `doSetUp` methods until we can get rid of PHPUnit < 8.

Reply:
I confirm: the trait has been designed so that only the signature of the method is changed, and not its body.